Bigtable/src/Exception/BigtableDataOperationException.php

```diff
@@ -17,14 +17,11 @@
 
 namespace Google\Cloud\Bigtable\Exception;
 
-use Google\Cloud\Core\Exception\GoogleException;
-
 /**
  * Exception thrown when a request fails.
  */
-class BigtableDataOperationException extends GoogleException
+class BigtableDataOperationException extends \Exception
```

Comment:
We are planning to not extend the exceptions in `Core` anymore, so for now just having this extend the base exception.